### ISSUE TITLE
Super expression must either be null or function, not undefined

### ISSUE THREAD

**Issue by jurajkrivda** [yelled3/react-native-google-static-map#3]
Include library

var GoogleStaticMap = require('react-native-google-static-map');

Usage

<GoogleStaticMap 
            style={styles.map} {...locationProps} 
            latitude={'32.064171'}
            longitude={'34.7748068'}
            zoom={13}
            size={{ width: 300, height: 550 }}
        />

**Comment by npomfret** [comment]
Same problem here.  Any ideas?

**Comment by jurajkrivda** [comment]
Try to use import statement
import GoogleStaticMap from 'react-native-google-static-map'

**Comment by npomfret** [comment]
It can be fixed by changing [GoogleStaticMap.js](https://github.com/yelled3/react-native-google-static-map/blob/master/GoogleStaticMap.js):

```
import {Image, PixelRatio} from 'react-native';
import React,{Component} from 'react';

```

I see theres a pull request pending.  For the moment I just copied the class into my own codebase.

**Comment by yelled3** [comment]
@npomfret @jurajkrivda fixed the issue in `v0.1.7`

**Comment by flexdhana** [comment]
I still get this issue “Super expression must either be null or a function, not object”  with 

import VideoPlayer from 'react-native-videoplayer'; (this is updated 11 months ago)

here is the versions from package.json

 "dependencies": {
    "react": "^15.1.0",
    "react-native": "^0.27.2",
    "react-native-animatable": "^0.6.1",
    "react-native-timer": "^1.1.2",
    "react-native-videoplayer": "^1.2.0"
  }

any suggestion?
